lib/addresses.js

```diff
@@ -184,12 +184,28 @@ AddressController.prototype.transformUtxo = function(utxo) {
 AddressController.prototype.multitxs = function(req, res, next) {
   var self = this;
 
-  this.node.getAddressHistory(req.addrs, true, function(err, txinfos) {
+  this.node.getAddressHistoryCount(req.addrs, {}, function(err, count) {
     if(err) {
       return common.handleErrors(err, res);
     }
 
-    res.jsonp(self.transformAddressHistoryForMultiTxs(txinfos));
+    var options = {
+      from: req.query.from || req.body.from || 0,
+      to: req.query.to || req.body.to || count
```

Comment:
Since we need the count for the totalItems field on the response anyway, it's easy to just put it there too.